src/panels/developer-tools/statistics/developer-tools-statistics.ts

```diff
@@ -116,26 +116,20 @@ class HaPanelDevStatistics extends SubscribeMixin(LitElement) {
       },
       actions: {
         title: "",
-        type: "overflow-menu",
+        label: localize("ui.panel.developer-tools.tabs.statistics.adjust_sum"),
+        type: "icon-button",
         template: (_info, statistic: StatisticsMetaData) =>
           statistic.has_sum
-            ? html`<ha-icon-overflow-menu
-                .hass=${this.hass}
-                .narrow=${this.narrow}
-                .items=${[
-                  {
-                    path: mdiSlopeUphill,
-                    label: localize(
-                      "ui.panel.developer-tools.tabs.statistics.adjust_sum"
-                    ),
-                    action: () =>
-                      showStatisticsAdjustSumDialog(this, {
-                        statistic: statistic,
-                      }),
-                  },
-                ]}
-                style="color: var(--secondary-text-color)"
-              ></ha-icon-overflow-menu>`
+            ? html`
+                <ha-icon-button
+                  .label=${localize(
+                    "ui.panel.developer-tools.tabs.statistics.adjust_sum"
+                  )}
+                  .path=${mdiSlopeUphill}
+                  .statistic=${statistic}
+                  @click=${this._showStatisticsAdjustSumDialog}
+                ></ha-icon-button>
+              `
             : html`<div style="width:48px"></div>`,
```

Comment:
I added this div because otherwise the column would disappear and the "FIX ISSUE" column wasn't aligned. I guess we can just throw this on the width of the column instead of hardcoding like this, maybe icon-button already does ?